apps/dataset-browser/src/app/[locale]/page.tsx

```diff
@@ -19,27 +38,53 @@ const filterKeysOrder: ReadonlyArray<keyof SearchResult['filters']> = [
   'licenses',
 ];
 
+export interface FacetMenuProps {
+  filters: SearchResult['filters'];
+  filterKeysOrder: ReadonlyArray<keyof SearchResult['filters']>;
+}
+
+function FacetMenu({filterKeysOrder, filters}: FacetMenuProps) {
+  const t = useTranslations('Home');
+
+  return (
+    <>
+      <SearchField />
+      {filterKeysOrder.map(
+        filterKey =>
+          !!filters[filterKey]?.length && (
+            <FilterSet
+              key={filterKey}
+              title={t(`${filterKey}Filter`)}
+              searchResultFilters={filters[filterKey]}
+              filterKey={filterKey}
+              testId={`${filterKey}Filter`}
+            />
+          )
+      )}
+    </>
+  );
+}
+
 interface Props {
   searchParams?: SearchParams;
 }
 
-// Revalidate the page
-export const revalidate = 0;
-
 export default async function Home({searchParams}: Props) {
   const searchOptions = fromSearchParamsToSearchOptions(searchParams ?? {});
   const sortBy = getClientSortBy({
     sortBy: searchOptions.sortBy,
     sortOrder: searchOptions.sortOrder,
   });
 
-  let hasError, searchResult;
+  let hasError, searchResultOrUndefined;
   try {
-    searchResult = await datasetFetcher.search(searchOptions);
+    searchResultOrUndefined = await datasetFetcher.search(searchOptions);
```

Comment:
The name of the variable suggests there is a 'search result' or an 'undefined'. But if I look at the return value of `search()`, there can only be a 'search result'?